src/components/HistoryTable/types.ts

```diff
@@ -1,12 +1,30 @@
 import { MainlineCommitsForHistoryQuery } from "gql/generated/types";
+import { Unpacked } from "types/utils";
+
+export type CommitRowType =
+  | {
+      type: rowType.FOLDED_COMMITS;
+      rolledUpCommits: Unpacked<
+        mainlineCommits["versions"]
+      >["rolledUpVersions"];
+      rowHeight: number;
+      date: Date;
+      selected: boolean;
+    }
+  | {
+      type: rowType.DATE_SEPARATOR;
+      date: Date;
+      rowHeight: number;
+      selected: boolean;
+    }
```

Comment:
Would it make sense/is it possible to exclude the `selected` field in the date separator type? we always designate this as `selected: false` which means it will never be selectable, so maybe we can omit this field?

Reply:
![image](https://user-images.githubusercontent.com/4605522/160713672-fff0a9d6-0932-4e54-9640-5cf87630ac61.png)
it breaks the typings :( 
VariantHistoryRow and taskHistoryRow have no way of knowing that a CommitRowType is a dateSeperator.